content/en/blog/2022/collector-builder-sample/index.md

```diff
@@ -0,0 +1,314 @@
+  process for setting up a custom collector to a few simple Make commands.
+
+- **Keep your Collector deployment up-to-date.** By incorporating custom
+  Collectors into your CI/CD pipeline, you can automate builds to ensure your
+  Collector stays current with the latest features and fixes. In this repo,
+  we'll show one way to do that with Cloud Build.
+
+Each of these represents part of the “Getting Started” process with
+OpenTelemetry Collector, so by identifying and consolidating these steps we hope
+to expedite and ease the experience down to a few clicks.
+
+## Build an OpenTelemetry Collector that suits your needs
+
+While there are
+[public Docker images for the Collector](https://hub.docker.com/r/otel/opentelemetry-collector-contrib/tags)
+published by the OpenTelemetry community, these images can be as large as 40MB.
+This is due to all of the
+[receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver),
+[processors](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor),
+and
+[exporters](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter)
+that are bundled into the image by default. With all of these default
+components, there is also the potential for security issues to arise, part of
+the reason why the
+[Collector maintainers recommend only enabling necessary components](https://github.com/open-telemetry/opentelemetry-collector/blob/5ab00fc/docs/security.md)
+in your Collector.
+
+The
+[collector-builder tool](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder)
+accelerates compiling stripped-down Collector images with a simple YAML config
+file. This file declares which components to include in the build, and
+collector-builder includes only those components (and nothing else). This is in
+contrast to a default Collector build, where many more components are included
+in the compiled binary (even if they are not being enabled in the Collector’s
+runtime configuration). This difference is how you can achieve improvements over
+public images that include many excess components, with extra dependencies
+adding weight and security burden. Now, those extra components can’t take up
+space because they aren’t even available in the custom build.
+
+To show how this works we start with a sample config file in this repository
+which has been pre-filled with a few of the most relevant OpenTelemetry
+components for GCP. However, we encourage you to modify that sample file to fit
+your use case.
+
+For example, the following build file includes only the OTLP receiver and GCP
+exporter, along with a logging exporter:
+
+```
+receivers:
+  - import: go.opentelemetry.io/collector/receiver/otlpreceiver
+    gomod: go.opentelemetry.io/collector v0.57.2
+
+exporters:
+  - import: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter
```

Comment:
Please use otlp exporters here (and in the line below): we made that case with other blog posts as well that their vendor-specific backend should not be mentioned in our blog to provide a common ground.